src/main/java/com/apuope/apuope_re/repositories/ConversationRepository.java

```diff
@@ -0,0 +1,91 @@
+package com.apuope.apuope_re.repositories;
+
+import com.apuope.apuope_re.dto.ConversationData;
+import com.apuope.apuope_re.dto.MessageData;
+import com.apuope.apuope_re.jooq.tables.Conversation;
+import com.apuope.apuope_re.jooq.tables.Message;
+import com.apuope.apuope_re.jooq.tables.records.ConversationRecord;
+import com.apuope.apuope_re.jooq.tables.records.MessageRecord;
+import org.jooq.DSLContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.List;
+
+@Repository
+public class ConversationRepository {
+    private final ZoneId TIMEZONE = ZoneId.of("Europe/Helsinki");
+
+    @Autowired
+    public ConversationRepository(){};
+
+    // Conversation
```

Comment:
Delete unneccessary comment.

Reply:
The idea with this and "// Message" was that because Conversation and Message are different tables but are used pretty much always together, it's reasonable to put them under same repository-file. Comments are only as an instruction for developer to see what is called, since ConversationRepository isn't only about conversation.

 I don't see these comments as unnecessary.